### PR TITLE
Up python version in Dockerfile

### PR DESCRIPTION
**Pull request by blablatdinov**

### CODE REVIEW

**Comment by blablatdinov** [comment]
@cclauss take a look

**Comment by cclauss** [comment]
### ChatGPT says…
The relative sizes of Docker images can vary depending on the specific configuration and content of each image. Generally, you can expect:

1.	Alpine:python3.13 - This is typically the smallest, often around 5-10 MB due to Alpine’s minimal base.
2.	Bookworm-slim:python3.13 - This image is usually larger than Alpine but smaller than the full Bookworm image, generally around 30-50 MB.
3.	Bookworm:python3.13 - This is the largest of the three, often exceeding 100 MB, as it includes more libraries and dependencies.

For the most accurate sizes, you can pull the images and check their sizes using docker images on your system.

**Comment by blablatdinov** [comment]
@cclauss I'm agree, my bad. Image changed

**Comment by cclauss** [comment]
% `docker pull python:3.13-slim`
% `docker pull python:3.13-alpine`
% `docker pull python:3.13-bookworm`
% `docker image ls | grep python`
```
python                              3.13-slim         751d8bece269   12 days ago     207MB
python                              3.13-alpine       c38ead8bcf52   12 days ago     72.9MB
python                              3.13-bookworm     feee4734fdc4   12 days ago     1.47GB
```

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM python:3.9-alpine
+FROM python:3.13-slim
```
Comment (cclauss) [inline Dockerfile]:
```suggestion
FROM python:3.13-alpine
```

**Review by cclauss — APPROVED** [review]
Done but archiving this repo in favor of https://github.com/astral-sh/ruff-action with the command `ruff check --select=PL --output-format=github`

**Pull request merged**